pkgs/top-level/aliases.nix

```diff
@@ -1476,6 +1476,7 @@ mapAliases ({
   xcbuild6Hook = throw "'xcbuild6Hook has been renamed to/replaced by 'xcbuildHook'"; # Added 2023-12-10
   xcodebuild6 = throw "'xcodebuild6' has been renamed to/replaced by 'xcodebuild'"; # Added 2023-12-10
   xdg_utils = xdg-utils; # Added 2021-02-01
+  xfce4-embed-plugin = throw "xfce4-embed-plugin has been removed, as it was broken"; # Added 2024-07-15
```

Comment:
I guess we want to add the alias in `pkgs/desktops/xfce/default.nix`?

Reply:
Do we want throws there? I thought removed those to not overwhelm hydra with error message reports (performance).

Reply:
There is an alias section already where this can be put